app/src/main/java/com/example/music_app/ui/helpers/RetrofitAuthHelper.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.music_app.ui.helpers
+
+import com.example.music_app.constants.BASE_URL
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitAuthHelper {
+    private const val baseUrl = BASE_URL
+
+    fun getInstance(): Retrofit {
```

Comment:
Why do you need this helper class? The creation logic could be moved to the `Factory` companion object of the `AuthService`
For instance
```
companion object Factory {
        private const val BASE_URL: String = "..."
        fun create(): SpotifyApi {
            val retrofit = Retrofit.Builder()
                .addConverterFactory(GsonConverterFactory.create())
                .baseUrl(BASE_URL)
                .build()
            return retrofit.create(SpotifyApi::class.java)
        }
```